src/librustdoc/clean/mod.rs

```diff
@@ -3680,6 +3755,9 @@ fn print_const(cx: &DocContext, n: &ty::Const) -> String {
                 inline::print_inlined_const(cx, def_id)
             }
         },
+        ConstValue::Param(..) => {
+            unimplemented!() // TODO(const_generics:rustdoc)
```

Comment:
Does this need special work? I would expect `fmt_const_val` below to "just work".

Reply:
Ah, that's true.